css/new.css

```diff
@@ -0,0 +1,137 @@
+body{
+	background-color: #f9f9f9;
+}
+
+.main-section{
+	width: 80%;
+	margin: 0 auto;
+	max-width: 1200px;
+}
+
+form{
+	margin-bottom: 20px;
+	display: inline-block;
+}
+
+.form-section-first{
+    display: flex;
+	justify-content: space-around;
+}
+
+.form-section-first input{
+	width: 31%;
+	border: 0;
+	padding-left: 10px;
+	height: 40px;
+	color: lightgrey;
+	font-size: 20px;
+	padding-right: 10px;
+
+
+}
+
+.form-section-first input::placeholder{
+	
+	border-bottom: 1px solid lightgrey;
+	color: lightgrey;
+
+}
+
+.form-section-first input:focus{
+	outline:none;
+}
+
+.form-section span{
+	font-size: 25px;
+	margin-left: 30px;
+	margin-bottom: 30px;
+}
+
+header{
+	margin-bottom: 100px;
+}
+
+fieldset, .form-section, .form-section-first{
+	margin-bottom: 30px;
+}
+
+table{
+	width: 100%;
+	border-collapse: collapse;
+	text-align: center;
+	margin-top: 10px;
+}
+
+textarea{
+	width: 100%;
+	resize: none;
+	height: 100px;
+	border: 2px solid darkgrey;
+	width: calc(100% - 25px);
```

Comment:
I love calc . For some reason my textarea boxes are with 25px more on width, no matter of the size of the screen. I guess this comes from padding+border. So I tought this could be one good solution.